packages/cashier/src/constants/transaction-status.tsx

```diff
@@ -87,6 +87,20 @@ export const getStatus = (
                 renderer: 'unsuccessful',
                 transaction_hash: localize('NA'),
             },
+            REVERTED: {
+                name: localize('Unsuccessful'),
+                description: localize(
+                    "Your withdrawal is unsuccessful. We've sent you an email with more information."
+                ),
+                renderer: 'unsuccessful',
+                transaction_hash: localize('NA'),
+            },
+            REVERTING: {
+                name: localize('In process'),
+                description: localize("We're processing your withdrawal."),
+                renderer: 'in-process',
+                transaction_hash: formatted_transaction_hash,
```

Comment:
@nada-deriv we don't have transaction hash in `REVERTING` status, but I'm not sure whether this could breaks FE or not as I noticed that it is being used in `VERIFIED` and `LOCKED` statuses, but we still don't have the hash available in those statuses.